### PR TITLE
React Cinema - YETFLIX

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    if (type === 'results') {
+      this.setState({ submittedQuery: query,
+                      page: 1,
+                      newSearchInProgress: true });
+      apiQuery = `http://www.omdbapi.com/?apikey=eee5954b&s=${query}&type=movie`;
+      this.fetchData(apiQuery, type);
+    }
+
+    else if (type === 'resultsInfinite') {
+      const {nextPageFetchInProgress} = this.state;
```
Comment (dmitrigrabov) [inline src/components/App.js:55]: nice

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+      apiQuery = `http://www.omdbapi.com/?apikey=eee5954b&s=${query}&type=movie`;
+      this.fetchData(apiQuery, type);
+    }
+  }
+
+  receiveMovie(movie, source) {
+    const apiQuery = `http://www.omdbapi.com/?apikey=eee5954b&i=${movie.imdbID}`;
+    this.fetchData(apiQuery, 'detail');
+    if (source === 'fromFavs') {
+      setTimeout(() => this.setState({ favsDisplayed: false }), 500);
```
Comment (dmitrigrabov) [inline src/components/App.js:76]: setTimeout feels awkward here. there is probably a better way of doing this. give me a shout to go over this if you want

**Comment by dmitrigrabov** [comment]
Great work!

File: src/components/App.js (hunk 2 above)
Comment (ghost) [reply src/components/App.js:76]: probably css transitions. was too lazy to deal with css on the weekend :)

**Pull request closed without merging**